docker-compose.yml

```diff
@@ -38,4 +46,21 @@ services:
       - api
     networks:
       - default
-    restart: always
\ No newline at end of file
+    logging:
+      driver: "syslog"
+      options:
+        syslog-address: "tcp://mozdef_logging_agent:514"
+    restart: always
+  mozdef_logging_agent:
+    build:
+      context: .
+      dockerfile: ./docker/mozdef_logging_agent
+    command: INSERT_COMMAND
+    environment:
+      - INSERT_SECRET_NAME_AND_DEFAULT_VALUE_EXAMPLE1
+      - INSERT_SECRET_NAME_AND_DEFAULT_VALUE_EXAMPLE2
```

Comment:
I suspec this is where we'll want to land whatever secrets we'll need to give the logging agent so it can authenticate to the mozdef pipeline.